.changeset/modern-frogs-roll.md

```diff
@@ -0,0 +1,24 @@
+---
+"@biomejs/biome": minor
```

Comment:
if this is a minor, it should be based on `next`

Reply:
Thanks! I forgot to push the change.